.github/ISSUE_TEMPLATE/2-deprecate-sgid-dataset.md

```diff
@@ -30,15 +30,19 @@ The replaced data is still accessible via our shelved policy in AGOL (a link to
 1. _~Strike~ out all items that do not apply._
 
 - [ ] Remove data from the Internal SGID (name, completed: `2021/00/00`)
+- [ ] Mark ArcGIS Online item as deprecated in preparation for future deletion (name, completed: `2021/00/00`)
+  - [ ] Add `Deprecated` to `AGOL_PUBLISHED_NAME` in `SGID.META.AGOLItems` (name, completed: `2021/00/00`)
```

Comment:
**issue**: this might cause cloudb to publish a new open sgid dataset and I'm not sure what the agol pallet will do

Reply:
Ah, didn't think about that.

Reply:
Solved with auditor changing the title without adjusting the name in the metatable.